### PR TITLE
doc: improve the multi-disk section on host page

### PR DESCRIPTION
**Pull request by Vicente-Cheng**
related issues: #244

### CODE REVIEW

File: docs/host/host.md [hunk 1]
```diff
@@ -63,10 +63,28 @@ Users can view and add multiple disks as additional data volumes from the host d
...
+Now, Harvester does not support adding specific partition as disk. Make sure your disk does not contain any partition or you could not find it at this stage.
+
+:::
+
+4. Select an additional raw block device to add as an additional data volume.
     - The `Force Formatted` option is required if the block device has never been force-formatted.
 
+![Force Format](/img/v1.1/host/force-format-disks.png)
+
+5. Last, you can check the newly added disk by **⋮ > Edit Config** again. You can also add the NODE/DISK tag this time (Describe detail in the next section).
```
Comment (vickyhella) [inline docs/host/host.md:84]:
```suggestion
5. Last, you can check the newly added disk by clicking **⋮ > Edit Config** again. You can also add the NODE/DISK tag this time (details are described in the next section).
```

**Review by vickyhella — COMMENTED** [review]
Just a small suggestion, the rest LGTM!

File: docs/host/host.md (hunk 1 above)
Comment (Vicente-Cheng) [reply docs/host/host.md]: done, thanks!

**Review by vickyhella — APPROVED** [review]

File: docs/host/host.md [hunk 2]
```diff
@@ -63,10 +63,28 @@ Users can view and add multiple disks as additional data volumes from the host d
...
+
+![Edit Config](/img/v1.1/host/edit-config.png)
+
+3. Select the **Storage** tab and click **Add Disk**.
+
+![Add Disks](/img/v1.1/host/add-disks.png)
+
+:::caution
+
+Now, Harvester does not support adding specific partition as disk. Make sure your disk does not contain any partition or you could not find it at this stage.
```
[the remark this answers is not in the document]
Comment (guangbochen) [reply docs/host/host.md:75]: Can u please help to double-check the deprecated version, thanks?

[the remark this answers is not in the document]
Comment (Vicente-Cheng) [reply docs/host/host.md:75]: Sure, I would like to

[the remark this answers is not in the document]
Comment (Vicente-Cheng) [reply docs/host/host.md:75]: I found the UI PR https://github.com/rancher/dashboard/pull/5297
That makes the user only use the whole disk.
And NDM also has some related PR for drop partition mountpoint.
So I thought that we do not support partition after v1.0.2.

File: docs/host/host.md [hunk 3]
```diff
@@ -63,10 +63,28 @@ Users can view and add multiple disks as additional data volumes from the host d
...
+
+![Edit Config](/img/v1.1/host/edit-config.png)
+
+3. Select the **Storage** tab and click **Add Disk**.
+
+![Add Disks](/img/v1.1/host/add-disks.png)
+
+:::caution
+
+As of Harvester v1.0.2, we no longer support adding partitions as additional disks. If you want to add it as an additional disk, be sure to reformat it properly first(e.g., using `fdisk`).
```
Comment (btat) [inline docs/host/host.md:75]:
```suggestion
As of Harvester v1.0.2, we no longer support adding partitions as additional disks. If you want to add it as an additional disk, be sure to reformat it properly first (e.g., using `fdisk`).
```

File: docs/host/host.md [hunk 4]
```diff
@@ -63,10 +63,28 @@ Users can view and add multiple disks as additional data volumes from the host d
...
+
+:::
+
+4. Select an additional raw block device to add as an additional data volume.
     - The `Force Formatted` option is required if the block device has never been force-formatted.
 
+![Force Format](/img/v1.1/host/force-format-disks.png)
+
+5. Last, you can click **⋮ > Edit Config** again to check the newly added disk. At meanwhile, you can also add the "Host/Disk" tag (details are described in the [next section](#storage-tags)).
```
Comment (btat) [inline docs/host/host.md:84]:
```suggestion
5. Last, you can click **⋮ > Edit Config** again to check the newly added disk. Meanwhile, you can also add the "Host/Disk" tag (details are described in the [next section](#storage-tags)).
```

**Review by btat — CHANGES_REQUESTED** [review]
Added a couple small suggestions.

File: docs/host/host.md (hunk 3 above)
Comment (Vicente-Cheng) [reply docs/host/host.md]: updated, thanks!

File: docs/host/host.md (hunk 4 above)
Comment (Vicente-Cheng) [reply docs/host/host.md]: updated!

**Review by guangbochen — APPROVED** [review]
LGTM, thanks.

**Review by bk201 — APPROVED** [review]
LGTM, thanks!

**Review by btat — APPROVED** [review]
LGTM

**Pull request merged**